rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -796,4 +799,26 @@ public Integer removeLockWhitelistAddress(Object[] args)
 
         return bridgeSupport.removeLockWhitelistAddress(rskTx, addressBase58);
     }
+
+    public Integer voteFeePerKbChange(Object[] args)
+    {
+        logger.trace("voteFeePerKbChange");
+
+        Coin feePerKb;
+        try {
+            feePerKb = Coin.valueOf(((BigInteger) args[0]).longValueExact());
+        } catch (Exception e) {
+            logger.warn("Exception in voteFeePerKbChange: {}", e.getMessage());
```

Comment:
small tweak, log exception and not message

Reply:
done!